.gitignore

```diff
@@ -138,11 +138,7 @@ project/metals.sbt
 ## Build Cache ##
 #################
 
-bazel-bin
-bazel-bazel
-bazel-enso
-bazel-out
-bazel-testlogs
+bazel-*
```

Comment:
I should note that build system still expects that the directory is named `bazel-enso`, but only in one place – when cleaning up after CI build.

Reply:
I see. Tried to address that, but the solution ends up being quite complex and I worry it might cause more issues with wrongly deleted files. I don't want to over-complicate the cleanup right now, so I left it as is. It works properly on CI where it's necessary, and locally right now it will leave one junction behind, which can be deleted manually if really needed.